layouts/partials/header.html

```diff
@@ -0,0 +1,21 @@
+{{ $current := . }}
+<nav class="navbar navbar-expand-md navbar-dark bg-dark fixed-top shadow-sm" id="navbar-main-menu">
```

Comment:
same thing as the theme file but changing `{navbar,bg}-light` to `{navbar-bg}-dark` plus some 🍬

Reply:
it has, but since the body is dark, it's difficult to see XD 

https://deploy-preview-104--bcneng.netlify.app/freelancers/

Reply:
I'll redesign the freelancers page somehow

Reply:
https://github.com/bcneng/website/issues/105

Reply:
Okey, gotcha! thank you @gonzaloserrano